### PR TITLE
[CloudPC] Add Supported Region region group parameter and add RPA box to CloudPCManagementService enum

### PR DESCRIPTION
**Pull request by Meadow201317**

### CODE REVIEW

**Comment by AshleyYangSZ** [comment]
@Meadow201317 Could you please help add the change log?
And update the PR title, since it contains two graph changes, 1 is regionGroup, 2 is adding rpaBox enum member to supportedSolution.

**Comment by Meadow201317** [comment]
> @Meadow201317 Could you please help add the change log? And update the PR title, since it contains two graph changes, 1 is regionGroup, 2 is adding rpaBox enum member to supportedSolution.

Updated, thx!

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 1]
```diff
@@ -23,12 +23,15 @@ Represents a supported region to establish an Azure network connection for Cloud
...
-|:---|:---|:---|
+| Property | Type | Description |
+|:---      |:---  |:---         |
 |displayName|String|The name for the supported region. Read-only.|
 |id|String|The unique identifier for the supported region. Read-only.|
 |regionStatus|[cloudPcSupportedRegionStatus](#cloudpcsupportedregionstatus-values)|The status of the supported region. Possible values are: `available`, `restricted`, `unavailable`, `unknownFutureValue`. Read-only.|
-|supportedSolution|[cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|The supported service or solution for the region. The possible values are: `windows365`, `devBox`, `unknownFutureValue`. Read-only.|
+|supportedSolution|[cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|The supported service or solution for the region. The possible values are: `windows365`, `devBox`, `rpaBox`,`unknownFutureValue`. Read-only.|
+|regionGroup|[cloudpcRegionGroup](#cloudpcregiongroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. Customer could select a regionGroup when provision a CloudPC,
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcsupportedregion.md:32]: Please update:
[cloudpcRegionGroup] => [cloudPcRegionGroup]
a CloudPC => a Cloud PC
the CloudPC => the Cloud PC

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -1,5 +1,31 @@
 {
     "changelog": [
+        {
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:3]: As for the changelog, please also add this one: 

{
          "Id": "dd8cc504-3861-450c-add5-763a11dd0266",
          "ApiChange": "Property",
          "ChangedApiName": "regionGroup",
          "ChangeType": "Addition",
          "Description": "Added the `regionGroup` property to the [cloudPcDomainJoinConfiguration](https://docs.microsoft.com/en-us/graph/api/resources/intune-cloudPcDomainJoinConfiguration?view=graph-rest-beta) resource.",
          "Target": "cloudPcDomainJoinConfiguration"
        },

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@Meadow201317 
1.Please resolve comments in this PR.
2.Please create a roadmap workitem and attach the workitem number in comment. This is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/).

**Comment by Meadow201317** [comment]
Here is the workitem: [Feature 15373](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15373): [DEVICES AND APPS][CLOUD PC] Add regionGroup to cloudPcSupportedRegion

**Review by AshleyYangSZ — APPROVED** [review]

**Comment by Meadow201317** [comment]
> @Meadow201317 1.Please resolve comments in this PR. 2.Please create a roadmap workitem and attach the workitem number in comment. This is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/).

Have resolved comments and the workitem has been attached

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 3]
```diff
@@ -23,12 +23,15 @@ Represents a supported region to establish an Azure network connection for Cloud
...
-|:---|:---|:---|
+| Property | Type | Description |
+|:---      |:---  |:---         |
 |displayName|String|The name for the supported region. Read-only.|
 |id|String|The unique identifier for the supported region. Read-only.|
 |regionStatus|[cloudPcSupportedRegionStatus](#cloudpcsupportedregionstatus-values)|The status of the supported region. Possible values are: `available`, `restricted`, `unavailable`, `unknownFutureValue`. Read-only.|
-|supportedSolution|[cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|The supported service or solution for the region. The possible values are: `windows365`, `devBox`, `unknownFutureValue`. Read-only.|
+|supportedSolution|[cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|The supported service or solution for the region. The possible values are: `windows365`, `devBox`, `rpaBox`,`unknownFutureValue`. Read-only.|
+|regionGroup|[cloudPcRegionGroup](#cloudpcregiongroup-values)|The logic geographic group this region belongs to. Multiple regions could belong to one region group. Customer could select a regionGroup when provision a Cloud PC,
```
Comment (AshleyYangSZ) [inline api-reference/beta/resources/cloudpcsupportedregion.md:32]: image
There're some format issue caused this grid shows incorrectly. Please help update.

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 3 above)
Comment (Meadow201317) [reply api-reference/beta/resources/cloudpcsupportedregion.md]: Fixed the format, thx!

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+                    "ChangeType": "Addition",
+                    "Description": "Added the **cloudPcRegionGroup** enumeration type.",
+                    "Target": "cloudPcRegionGroup"
+                },
+                {
+                    "Id": "dd8cc504-3861-450c-add5-763a11dd0266",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "regionGroup",
+                    "ChangeType": "Addition",
+                    "Description": "Added the regionGroup property to the cloudPcDomainJoinConfiguration resource.",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:26]: 1. Add to `cloudPcSupportedRegion`, not `cloudPcDomainJoinConfiguration `. 
2. Add please add the link of `cloudPcSupportedRegion `here. You can refer to other examples.

Comment (Meadow201317) [reply changelog/Microsoft.CloudManagedDesktop.json]: Updated, thx!

**Review by RuiHou105 — APPROVED** [review]

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 5]
```diff
@@ -39,6 +40,32 @@ Represents a supported region to establish an Azure network connection for Cloud
...
+|:-------------------|:-----------------------------------------------------------------------------------|
+| default            | The region belongs to the default region group.                                    |
+| australia          | The region belongs to the region group: Australia.                                 |
+| canada             | The region belongs to the region group: Canada.                                    |
+| usCentral          | The region belongs to the region group: Central US.                                |
+| usEast             | The region belongs to the region group: East US.                                   |
+| usWest             | The region belongs to the region group: West US.                                   |
+| france             | The region belongs to the region group: France.                                    |
+| germany            | The region belongs to the region group: Germany.                                   |
+| europeUnion        | The region belongs to the region group: European Union.                            |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcsupportedregion.md:55]: According to the api.md file, the correct member name for europeUnion is europe. On the other hand, the live schema contains europeUnion. Is the CSDL/api.md file outdated? Please, verify the name of this member and update the value if needed.

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 6]
```diff
@@ -39,6 +40,32 @@ Represents a supported region to establish an Azure network connection for Cloud
...
+| asia               | The region belongs to the region group: Asia.                                      |
+| india              | The region belongs to the region group: India.                                     |
+| southAmerica       | The region belongs to the region group: South America.                             |
+| usGovernment       | The region belongs to the region group: US Government.                             |
+| usGovernmentDod    | The region belongs to the region group: US Government Department of Defense (DOD). |
+| unknownFutureValue | Evolvable enumeration sentinel value. Do not use.                                  |
+| norway             | The region belongs to the region group: Norway.                                    |
+| switzerland        | The region belongs to the region group: Switzerland.                               |
+| southKorea         | The region belongs to the region group: South Korea.                               |
+| uae                | The region belongs to the region group: UAE.                                       |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcsupportedregion.md:67]: According to the CSDL file, uae is a hidden value. Hidden values shouldn't be documented. Please, remove this enum member (uae) from this table.

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 7]
```diff
@@ -39,6 +40,32 @@ Represents a supported region to establish an Azure network connection for Cloud
...
+| usEast             | The region belongs to the region group: East US.                                   |
+| usWest             | The region belongs to the region group: West US.                                   |
+| france             | The region belongs to the region group: France.                                    |
+| germany            | The region belongs to the region group: Germany.                                   |
+| europeUnion        | The region belongs to the region group: European Union.                            |
+| unitedKingdom      | The region belongs to the region group: United Kingdom.                            |
+| japan              | The region belongs to the region group: Japan.                                     |
+| asia               | The region belongs to the region group: Asia.                                      |
+| india              | The region belongs to the region group: India.                                     |
+| southAmerica       | The region belongs to the region group: South America.                             |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcsupportedregion.md:60]: According to the CSDL file, the value "euap" should come after "southAmerica". Please, document the missing enum member.

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 8]
```diff
@@ -39,6 +40,32 @@ Represents a supported region to establish an Azure network connection for Cloud
...
+| usGovernment       | The region belongs to the region group: US Government.                             |
+| usGovernmentDod    | The region belongs to the region group: US Government Department of Defense (DOD). |
+| unknownFutureValue | Evolvable enumeration sentinel value. Do not use.                                  |
+| norway             | The region belongs to the region group: Norway.                                    |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcsupportedregion.md:64]: The CSDL file doesn't contain any entry for the enum members "norway", "switzerland", and "southKorea". However, I see that they are already part of the live schema. Is the CSDL outdated?

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 9]
```diff
@@ -23,12 +23,13 @@ Represents a supported region to establish an Azure network connection for Cloud
 
 ## Properties
 
-|Property|Type|Description|
-|:---|:---|:---|
+| Property | Type | Description |
+|:---      |:---  |:---         |
 |displayName|String|The name for the supported region. Read-only.|
 |id|String|The unique identifier for the supported region. Read-only.|
+|regionGroup|[cloudPcRegionGroup](#cloudpcregiongroup-values)|The geographic group this region belongs to. Multiple regions can belong to one region group. For example, the Europe region group contains the Northern Europe and Western Europe regions. A customer can select a region group when provisioning a Cloud PC. The Cloud PC will be put under one of the regions under the group based on resource status. Possible values are: `default`, `australia`, `canada`, `usCentral`, `usEast`, `usWest`, `france`, `germany`, `europeUnion`, `unitedKingdom`, `japan`, `asia`, `india`, `southAmerica`, `euap`, `usGovernment`, `usGovernmentDod`, `norway`, `switzerland`，`southKorea`, `unknownFutureValue`. Read-only.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcsupportedregion.md:30]: "The Cloud PC will be put under one of the regions under the group based on resource status." Unfortunately, I don't understand this sentence. It this sentence connected to the previous sentence "A customer can select..." What do you mean by resource status - the status of the resource group in Azure? Please, rephrase this sentence to provide more clarity.

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 5 above)
Comment (Meadow201317) [reply api-reference/beta/resources/cloudpcsupportedregion.md:55]: We have update the CSDL file to EuropeUnion.

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 6 above)
Comment (Meadow201317) [reply api-reference/beta/resources/cloudpcsupportedregion.md:67]: Got it, thx!

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 7 above)
Comment (Meadow201317) [reply api-reference/beta/resources/cloudpcsupportedregion.md:60]: Sorry for missing it.

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 8 above)
Comment (Meadow201317) [reply api-reference/beta/resources/cloudpcsupportedregion.md:64]: Here is the latest CSDL file ref: [Pull Request 11933](https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/11933): [Cloud PC] Add Norway, Switzerland, SouthKorea and Qatar in CloudPcRegionGroup

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 9 above)
Comment (Meadow201317) [reply api-reference/beta/resources/cloudpcsupportedregion.md:30]: Thanks for your comment. The sentence "The Cloud PC will be put under one of the regions under the group based on resource status." is connected to the "A customer can select..." which means a customer need to choose a region group but finally where the cloudPC is placed is determined by the region's resource capacity. If customer choose region group X, region A and region B belong to region group X and region A has more quota than region B, we will put the cloud PC in region A.

Comment (JarbasHorst) [reply api-reference/beta/resources/cloudpcsupportedregion.md]: Thanks for the clarification :)

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for a final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**